dvc/config_schema.py

```diff
@@ -154,6 +154,7 @@ class RelPath(str):
                     "grant_read_acp": str,
                     "grant_write_acp": str,
                     "grant_full_control": str,
+                    "cache_regions": bool,
```

Comment:
What are the downsides of enabling this? What I'm wondering is why don't we just enable this by default?

Reply:
It is enabled by default, though this provides ability for our users to disable it.

Reply:
@isidentical Ah right, then why do we need to grant an ability to disable it?

Reply:
Since this is an experimental feature I thought it would be nice to give users a chance to disable it.